.yamllint.yaml

```diff
@@ -4,7 +4,7 @@ yaml-files:
   - "*.yml"
 
 ignore:
-  - charts/nginx-gateway-fabric/templates
+  - charts/nginx-gateway-fabric/
```

Comment:
question: isn't this the same location we store our charts? Aren't those the same charts we are linting?

Reply:
yeah we ignore it here in `.yamllint.yaml` which uses `yamllint` but we lint them with `chart-testing` that has the config in `.ct.yaml`